--- Makefile.origin	2021-01-18 22:10:46.716025341 +0200
+++ Makefile	2021-01-18 22:12:50.850488388 +0200
@@ -24,6 +24,7 @@
 VERSION=$(shell cat ../VERSION)
 DESTDIR?=/usr
 PREFIX?=/local
+PWD=`pwd`
 
 LDCONFIG?=ldconfig
 
@@ -37,7 +38,7 @@
 #DEBUG	= -g -O0
 DEBUG	= -O2
 CC	?= gcc
-INCLUDE	= -I.
+INCLUDE	= -I. -I$(PWD)/../wiringPi
 DEFS	= -D_GNU_SOURCE
 CFLAGS	= $(DEBUG) $(DEFS) -Wformat=2 -Wall -Winline $(INCLUDE) -pipe -fPIC $(EXTRA_CFLAGS)
 
