--- Makefile.origin	2021-01-18 22:10:46.736025093 +0200
+++ Makefile	2021-01-18 22:17:41.798895043 +0200
@@ -25,6 +25,7 @@
 
 DESTDIR?=/usr
 PREFIX?=/local
+PWD=`pwd`
 
 ifneq ($V,1)
 Q ?= @
@@ -33,10 +34,10 @@
 #DEBUG	= -g -O0
 DEBUG	= -O2
 CC	?= gcc
-INCLUDE	= -I$(DESTDIR)$(PREFIX)/include
+INCLUDE	= -I$(PWD)/../wiringPi -I$(PWD)/../devLib
 CFLAGS	= $(DEBUG) -Wall -Wextra $(INCLUDE) -Winline -pipe $(EXTRA_CFLAGS)
 
-LDFLAGS	= -L$(DESTDIR)$(PREFIX)/lib
+LDFLAGS	= -L$(PWD)/../wiringPi -L$(PWD)/../devLib
 LIBS    = -lwiringPi -lwiringPiDev -lpthread -lrt -lm -lcrypt
 
 # May not need to  alter anything below this line
